insights/config/specs.py

```diff
@@ -66,8 +66,8 @@
     "df_-li"                    : CommandSpec("/bin/df -li"),
     "dig"                       : CommandSpec("/usr/bin/dig +dnssec . DNSKEY"),
     "dirsrv"                    : SimpleFileSpec("etc/sysconfig/dirsrv"),
-    "dirsrv_access"             : PatternSpec("var/log/dirsrv/.*/access"),
-    "dirsrv_errors"             : PatternSpec("var/log/dirsrv/.*/errors"),
+    "dirsrv_access"             : PatternSpec("var/log/dirsrv/.*/(access|access\.2.*)"),
+    "dirsrv_errors"             : PatternSpec("var/log/dirsrv/.*/(errors|errors\.2.*)"),
```

Comment:
Would you please to update the corresponding items in `insights/specs_default.py` for the master (3.x) branch? Thanks.

Reply:
HI,

thanks a lot. When is this file used so as I could test it. Or is it used first before parsing config/specs.py ?

thanks.

German

Reply:
The '3.x' contains many new features but it is not released yet. The "insights/specs_default.py" and "insights/specs.py" are  the new styles spec files. And to keep compatibility with the current '1.x', we need to edit both the new style in "insights/specs_default.py" and "insights/specs.py" and the old one in "insights/config/specs.py".

You can test the updated spec written in new style by referring to corresponding step in https://github.com/RedHatInsights/insights-core/blob/master/notebooks/Datasource%20Registry.ipynb

Reply:
The new specs use `glob()`, so they should be written as:

```python
     "dirsrv_access"             : PatternSpec("var/log/dirsrv/*/access", "var/log/dirsrv/*/access.2*)"),
     "dirsrv_errors"             : PatternSpec("var/log/dirsrv/.*/errors", "var/log/dirsrv/*/errors.2*)"),
```